docs/content/development/extensions-core/s3.md

```diff
@@ -18,6 +18,18 @@ S3-compatible deep storage is basically either S3 or something like Google Stora
 |`druid.s3.secretKey`||S3 secret key.|Must be set.|
 |`druid.storage.bucket`||Bucket to store in.|Must be set.|
 |`druid.storage.baseKey`||Base key prefix to use, i.e. what directory.|Must be set.|
+|`druid.storage.storageDirectory`||Directory for storing segments.|Must be set.|
+
+
+Example conf:
+```json
+## s3
+druid.s3.accessKey=<YOURACCESSKEY TO ACCESS S3 BUCKET>
+druid.s3.secretKey=<YOURSECRETKEY TO ACCESS S3 BUCKET>
```

Comment:
This is a `PasswordProvider` as of https://github.com/druid-io/druid/pull/4613 so you don't actually need a plain text secret in your configurations.